nanopub/client.py

```diff
@@ -62,40 +67,18 @@ def find_nanopubs_with_text(self, text: str, max_num_results: int = 1000):
         """
         if len(text) == 0:
             return []
-
+        endpoint = 'find_signed_nanopubs_with_text'
```

Comment:
We actually never use the 'normal' find_nanopubs_with_text endpoint anymore.

Reply:
That's fine. I don't think unsigned nanopubs are really useful, given the whole point of the infrastructure being to assign authorship/reputation to assertions.